.github/workflows/ami-release-nix.yml

```diff
@@ -141,7 +141,7 @@ jobs:
     
       - name: Upload nix flake revision to s3 prod
         run: |
-          aws s3 cp /tmp/pg_binaries.tar.gz s3://${{ secrets.PROD_ARTIFACTS_BUCKET }}/upgrades/postgres/supabase-postgres-${{ steps.process_release_version.outputs.version }}/20.04.tar.gz
+          aws s3 cp /tmp/pg_binaries.tar.gz s3://${{ secrets.PROD_ARTIFACTS_BUCKET }}/upgrades/postgres/supabase-postgres-${{ steps.process_release_version.outputs.version }}/24.04.tar.gz
```

Comment:
Can these version changes be deduplicated?

Reply:
@steve-chavez we can certainly address this. But as @pcnc had discussed in slack thread, these are currently tied to infrastructure funcationality. So, instead of trying to make those changes as a part of the upgrade to ubuntu 24.04 PR, let's create another PR where we can introduce those changes and test and make sure nothing else breaks. We should not try to accomplish that in this PR.